test/fixtures/temp-1000/file2.js

```diff
@@ -0,0 +1 @@
+original
```

Comment:
@sokra Oops, thanks. Any reason not to add a pattern to `.gitignore` for these? As well as `test/statsCases/*/actual.txt`?